src/table/utils/selections-utils.ts

```diff
@@ -129,8 +129,11 @@ export const getSelectedRows = (
   if (evt.shiftKey && key.includes('Arrow')) {
     selectedRows[qElemNumber] = rowIdx;
     // add the next or previous cell to selectedRows, based on which arrow is pressed
-    selectedRows[key === 'ArrowDown' ? cell.nextQElemNumber : cell.prevQElemNumber] =
-      key === 'ArrowDown' ? rowIdx + 1 : rowIdx - 1;
+    if (key === 'ArrowDown') {
+      selectedRows[cell.nextQElemNumber as number] = rowIdx + 1;
+    } else {
+      selectedRows[cell.prevQElemNumber as number] = rowIdx - 1;
+    }
```

Comment:
This was cleaner when  you had to add the casting + one less check for key === 'ArrowUp'

Reply:
we can remove {}
```
if (key === 'ArrowDown') selectedRows[cell.nextQElemNumber as number] = rowIdx + 1;
else selectedRows[cell.prevQElemNumber as number] = rowIdx - 1;
```

Reply:
hmm yeah more compact but I would say it hurts readability, either or for me